auth/client/src/main/java/org/wildfly/security/auth/client/DefaultEncryptedExpressionContextProvider.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2024 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.security.auth.client;
+
+import static java.security.AccessController.doPrivileged;
+
+import java.security.PrivilegedAction;
+
+/**
+ * A lazily-initialized holder for the default encrypted expression context.
+ * If an error occurs setting up the default identity
```

Comment:
"default identity context" -> This is likely a copy/paste from something else, should be reworded

Reply:
fixed!